person.rb

```diff
@@ -10,39 +10,37 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   def correct_name
     @name
   end
 
   def can_use_services?
-    of_age
+    of_age? || @parent_permission
   end
 
   def parent_permission?
     @parent_permission
   end
 
+  # def rental
+  #   if @rental.empty?
+  #     "This person doesn't have any books."
+  #   else
+  #     @rental.map { |rental| rental.book.title }.join(', ')
+  #   end
+  # end
+
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+    @rental << rental
```

Comment:
- [ ] You did well making the changes are requested by the previous reviewer. However, I notice on line 38 you have an undefined variable `rental`. Perhaps you wanted to do something like this 👇🏾 

```ruby
 def add_rental(book, date)
    rental = Rental.new(date, book, self) # defined the variable rental here. 
    @rental << rental
  end
```

Please fix this

Reply:
yeh thanks for it 
🙏🙏🙏💪💪